docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+services:
+  rabbitmq:
+    image: 'rafakato/rabbitmq-delayed-message-exchange'
```

Comment:
Use the `rabbitmq:3.6.10-alpine`. We haven't determined if delayed messages are necessary yet.